File: Java/1-50/6.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int jil;
        String x = "";

        jil = Integer.parseInt(scanner.nextLine());

        if (jil < 10) {
            x = "000";
        } else if (jil < 100) {
            x = "00";
        } else if (jil < 1000) {
            x = "0";
        }

        if (jil % 400 == 0 || (jil % 4 == 0 && jil % 100 != 0)) {
            x = "12/09/" + x + jil;
        } else {
            x = "13/09/" + x + jil;
        }

        System.out.println(x);
    }
}

File: Kotlin/1-50/6.kt
import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)
    val jil = scanner.nextInt()

    var x = ""

    when {
        jil < 10 -> x = "000"
        jil < 100 -> x = "00"
        jil < 1000 -> x = "0"
    }

    x = if (jil % 400 == 0 || (jil % 4 == 0 && jil % 100 != 0)) {
        "12/09/$x$jil"
    } else {
        "13/09/$x$jil"
    }

    println(x)
}
